server/caching.js

```diff
@@ -1,4 +1,11 @@
 /**
+ * How Quintype handles caching
+ * - Whenever a collection/story/author is created or updated, the platform does cache/purging through CDN.
+ * - When a collection/story/author is created, the request body tag gets affected, then the frontend gets updated with the latest data with the help of cache-keys.
+ *
+ * How cache keys are created
+ * - Cache keys are generated in `quintype-node-framework` (server/caching.js). We have explained how cache keys are generated for each page type
```

Comment:
we can give reference to `cache keys are generated` docs if we have.

Reply:
it is in the same file